benchmarks/GraphBLAS-sharp.Benchmarks/Algorithms/BFS.fs

```diff
@@ -113,10 +113,12 @@ type WithoutTransferBenchmark<'elem when 'elem : struct>(
     override this.GlobalSetup() =
         this.ReadMatrix()
         this.LoadMatrixToGPU()
+        this.Processor.PostAndReply(Msg.MsgNotifyMe)
```

Comment:
How much is it necessary to duplicate this? Is it possible to make an extension?

Reply:
I tried to make extension for specialization of generic `MailboxProcessor` and failed, as far as I know it is possible only for static members and static method would look not much better than this

Reply:
Ok. Seems like that. Than maybe just 
```
module Queue = 
    let block (q: MailboxProcessor<Msg>) = q.PostAndReply(Msg.MsgNotifyMe)
```
 in GraphBlas-sharp.Backend.Objects

Reply:
I was told that this may work.
```
[<System.Runtime.CompilerServices.Extension>] // На F# 8 можно убрать.
type MailboxProcessorExts =
    [<System.Runtime.CompilerServices.Extension>]
    static member Block(this : MailboxProcessor<int>) =
        this.Post 42
```
We can choose!